### PR TITLE
feat: Implement delete member

### PR DESCRIPTION
**Pull request by levansuper**

### CODE REVIEW

File: src/services/routes/member.routes.ts [hunk 1]
```diff
@@ -115,5 +115,34 @@ export default function registerRoutes(
...
+
+      const getMemberResult = await getMember(orgName, memberId, serviceOptions);
+      if (!getMemberResult.didSucceed) {
+        await reply.code(RESPONSE_CODE_BY_PROBLEM[getMemberResult.reason]).send({
+          type: getMemberResult.reason,
+        });
+        return;
+      }
+
+      await deleteMember(orgName, serviceOptions);
```
Comment (gnarea) [inline src/services/routes/member.routes.ts:141]:
```suggestion
      await deleteMember(memberId, serviceOptions);
```

File: src/testUtils/stubs.ts [hunk 2]
```diff
@@ -5,3 +5,4 @@ export const NON_ASCII_AWALA_ENDPOINT = 'はじめよう.みんな';
 export const MEMBER_NAME = 'TEST_MEMBER_NAME';
 export const NON_ASCII_MEMBER_NAME = 'テスト ユーザー';
 export const MEMBER_EMAIL = 'membet@example.com';
+export const MEMBER_ID = '6424ad273f75645b35f9ee79';
```
Comment (gnarea) [inline src/testUtils/stubs.ts:8]: I'd call this something like `MEMBER_API_ID`. "Member id" is a different thing in VeraId.

**Review by gnarea — CHANGES_REQUESTED** [review]
Thanks!

File: src/testUtils/stubs.ts (hunk 2 above)
Comment (levansuper) [reply src/testUtils/stubs.ts:8]: Maybe MEMBER_MONGO_ID?

Comment (gnarea) [reply src/testUtils/stubs.ts]: Agreed. That's more accurate.

**Review by gnarea — APPROVED** [review]

File: src/testUtils/stubs.ts (hunk 2 above)
Comment (gnarea) [reply src/testUtils/stubs.ts]: I just approved the PR just in case, but feel free to use that name and I'll re-approve it.

**Review by gnarea — APPROVED** [review]
Thanks!

**Pull request merged**